packages/Webkul/Admin/src/Resources/assets/sass/partials/_default.scss

```diff
@@ -1022,4 +1022,18 @@ body {
             }
         }
     }
+}
+
+@media only screen and (min-width: 280px) and (max-width: 420px) {
+    .grid-container {
+        .grid-top {
+            .datagrid-filters {
+                .filter-left {
+                    div {
```

Comment:
In fact, there is an extra `div` here, which if you remove it, you get the same result without any new CSS: 

![image](https://user-images.githubusercontent.com/4408379/195358758-b2eb4cbd-2b83-4af8-a61f-bb1d5ab55123.png)

Reply:
However we can move `.filter-left` to the component itself as a new prop eg.